Index: src-mpi/Makefile.vanilla
===================================================================
--- src-mpi.orig/Makefile.vanilla
+++ src-mpi/Makefile.vanilla
@@ -15,8 +15,8 @@ DO_MPI = ON
 ### specify paths to include files that the compiler can't find on its
 ### own.  If you need any -L or -l switches to get C standard libraries
 ### (such as -lm for the math library) put them in C_LIB.
-CC = mpicc
-CFLAGS = -std=c99
+CC = smpicc
+CFLAGS = -D_GNU_SOURCE
 OPTFLAGS = -g -O5
 INCLUDES = 
 C_LIB = -lm
