Index: AMG/Makefile.include
===================================================================
--- AMG.orig/Makefile.include
+++ AMG/Makefile.include
@@ -23,8 +23,7 @@
 
 # set the compiler here
 #CC = gcc
-CC = mpicc
-#CC = mpixlc_r
+CC = smpicc
 
 # set compile flags here
 # MPI only:
@@ -40,14 +39,13 @@ CC = mpicc
 #   add '-DHYPRE_BIGINT'  to 'INCLUDE_CFLAGS'
 #INCLUDE_CFLAGS = -O2
 #INCLUDE_CFLAGS = -g -O2 -DHYPRE_SEQUENTIAL=1
-#INCLUDE_CFLAGS = -g -DTIMER_USE_MPI 
+INCLUDE_CFLAGS = -g -O2 -D__USE_XOPEN -D_GNU_SOURCE #-DTIMER_USE_MPI 
 #INCLUDE_CFLAGS = -g -DTIMER_USE_MPI -DHYPRE_USING_OPENMP
 #INCLUDE_CFLAGS = -O2 -DTIMER_USE_MPI -DHYPRE_USING_OPENMP -DHYPRE_BIGINT -fopenmp
-INCLUDE_CFLAGS = -O2 -DTIMER_USE_MPI -DHYPRE_USING_OPENMP -fopenmp -DHYPRE_HOPSCOTCH -DHYPRE_USING_PERSISTENT_COMM -DHYPRE_BIGINT
+#INCLUDE_CFLAGS = -O2 -DTIMER_USE_MPI -DHYPRE_USING_OPENMP -fopenmp -DHYPRE_HOPSCOTCH -DHYPRE_USING_PERSISTENT_COMM -DHYPRE_BIGINT
 #INCLUDE_CFLAGS = -O2 -DTIMER_USE_MPI -DHYPRE_USING_OPENMP -qsmp=omp -qmaxmem=-1 -DHYPRE_HOPSCOTCH -DHYPRE_USING_PERSISTENT_COMM -DHYPRE_BIGINT
 
 # set link flags here
 #INCLUDE_LFLAGS = -lm -qsmp=omp
-INCLUDE_LFLAGS = -lm -fopenmp
-#INCLUDE_LFLAGS = -lm 
-
+#INCLUDE_LFLAGS = -lm -fopenmp
+INCLUDE_LFLAGS = -lm 
\ No newline at end of file
