Index: miniSMAC2D/initia.f
===================================================================
--- miniSMAC2D.orig/initia.f
+++ miniSMAC2D/initia.f
@@ -169,9 +169,9 @@ c ... use default values for airfoil
         underr = 0.01D0           ! under-relaxation parameter, 0 < underr < 1
         write_plot3d_q_file=.false. ! do not write plot3d q file, may be big
         bcmain_directory = '.'
-        bcmain_filename = 'bcmain.dat_original'
+        bcmain_filename = 'bcmain.dat_original_7kx7k'
         xygrid_directory = '.'
-        xygrid_filename = 'xy.dat_original'
+        xygrid_filename = 'xy.dat_original_7kx7k'
         print*
         print*, 'DEFAULT AIRFOIL VALUES ARE BEING USED FOR THIS RUN:'
         if (DEBUG) then
