Index: ziatest/ziaprobe.c
===================================================================
--- ziatest.orig/ziaprobe.c
+++ ziatest/ziaprobe.c
@@ -50,7 +50,7 @@ int main(int argc, char* argv[])
     start_sec = strtol(argv[1], NULL, 10);
     start_usec = strtol(argv[2], NULL, 10);
     
-    MPI_Init(NULL, NULL);
+    MPI_Init(&argc, &argv);
     MPI_Comm_rank(MPI_COMM_WORLD, &rank);
     MPI_Comm_size(MPI_COMM_WORLD, &size);
 
