Index: msgrate/Makefile
===================================================================
--- msgrate.orig/Makefile
+++ msgrate/Makefile
@@ -21,9 +21,9 @@
 
 RELEASE_VERSION = 1.0
 
-#CC = mpicc
-CC = cc
-CPPFLAGS = -DVERSION=$(RELEASE_VERSION)
+CC = smpicc
+# CC = cc
+CPPFLAGS = #-DVERSION=$(RELEASE_VERSION)
 CFLAGS = -O3
 
 msgrate: msgrate.o
@@ -39,4 +39,4 @@ dist:
 	chmod 644 msgrate-$(RELEASE_VERSION)/$$file ; \
 	done)
 	tar czf msgrate-$(RELEASE_VERSION).tar.gz msgrate-$(RELEASE_VERSION)
-	rm -rf msgrate-$(RELEASE_VERSION)
+	rm -rf msgrate-$(RELEASE_VERSION)
\ No newline at end of file
